### ISSUE TITLE
[Global] Animation persists for reduce motion

### ISSUE THREAD

**Issue by svinkle** [CovidShield/mobile#50]
## Issue summary

The illustrative animations persist when the user has set their Accessibility settings to include the Reduce Motion option as `true`. The current animation could cause issues for folks with a Vestibular disorder who are susceptible to motion sickness. Animation can also be very distracting for people with a cognitive impairment or who are neurodivergent.

<details>
    <summary>Screenshots</summary>

<img width="373" alt="Screen Shot 2020-05-27 at 9 51 23 AM" src="https://user-images.githubusercontent.com/1392632/83031019-14595900-a002-11ea-9010-12cb5ec4d569.png">
<img width="372" alt="Screen Shot 2020-05-27 at 9 51 35 AM" src="https://user-images.githubusercontent.com/1392632/83031023-158a8600-a002-11ea-8bbd-34fbad59410c.png">

</details>

## Steps to reproduce

1. Enable [Reduce Motion in iOS](https://support.apple.com/en-ca/HT202655) (or Android)
2. Load the app into the iOS simulator
3. Review animated illustrations

### Expected

- Animations to run less frequently.

### Actual

- Animations run at regular iterations.

## Recommendation

When Reduce Motion setting is enabled, reduce the frequency of each animated illustration. This will help to provide a less distracting reading experience. This may be possible by using the [`react-reduce-motion` package](https://www.npmjs.com/package/react-reduce-motion).

Optionally, consider implementing a Pause Animation `button` in order to pause all animation throughout the app.

## Specifications

- **Component:** CovidShield [Global]
- **WCAG Principle:** Operable
- **WCAG SC:** 2.3.1 Three Flashes or Below Threshold
- **Severity:** Medium
- **Effort:** Unknown

**Comment by kpeatt** [comment]
I think we should be able to disable all animation when reduce animation is set. None of it is required. Maybe we can treat the tutorial animations as play once?

**Comment by svinkle** [comment]
@kpeatt Playing once sounds good. If the user wanted to consume the animation again they'd simply reopen the view.
